server/util.go

```diff
@@ -152,6 +152,7 @@ func AddCommands(
 		client.LineBreak,
 		tendermintCmd,
 		ExportCmd(ctx, cdc, appExport),
+		GenesisPortCmd(ctx, cdc),
```

Comment:
Not a fan of this TBH

Reply:
do you mean it shouldn't live under `gaiad` ?

Reply:
Correct.

Reply:
updated